OctoPrintLib/Main Classes/OctoprintFileOperation.cs

```diff
@@ -33,9 +33,11 @@ public OctoprintFileOperation(OctoprintServer server) : base(server)
         /// <param name="localDownloadPath">Full path (with</param>
         public async Task<bool> DownloadFileAsync(string remoteLocation, string localDownloadPath)
         {
+            if (System.IO.File.Exists(localDownloadPath))
```

Comment:
This wasn't the place iwanted to have the check. The check should be in the websocket handler in the startup.cs. Please remove this here. When we manually download a file it should be overwritten.